### PR TITLE
Adding::Glossary for Unit-1

### PR DESCRIPTION
**Pull request by lucifermorningstar1305**
Added the Unit 1 glossary. As discussed in https://github.com/huggingface/deep-rl-class/issues/32#issuecomment-1341031130. 
Integrated mine + @daspartho + @misza222 key-concepts summarization into the `.units/unit1/glossary.mdx` file.

### CODE REVIEW

File: .units/unit1/glossary.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- **Optimal Policy**: Policy that **maximizes** the **expected return** when an agent acts according to it. It is learned through *training*.
+
+### Policy-based Methods:
+- An approach to solving RL problems.
+- In this method, the Policy is learned directly. 
+- Will map each state to the best corresponding action at that state. Or a probability distribution over the set of possible actions at that state.
+
+### Value-based Methods:
+- Another approach to solving RL problems.
+- Here, instead of training a policy, we train a **value function** that maps each state to the expected value of being in that state.
```
Comment (simoninithomas) [inline .units/unit1/glossary.mdx:40]: Awesome work !

I would add two things:

Contributions are welcomed 🤗
If you want to improve the course, you can [open a Pull Request.](https://github.com/huggingface/deep-rl-class/pulls)

This glossary was made possible thanks to:

- [@lucifermorningstar1305](https://github.com/lucifermorningstar1305)
- [@daspartho](https://github.com/daspartho)
- [@misza222](https://github.com/misza222)

Comment (lucifermorningstar1305) [reply .units/unit1/glossary.mdx:40]: Hi Thomas, 

I have added the Contributions section, as you requested.

File: .units/unit1/glossary.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Glossary [[glossary]]
+
+### Markov Property
```
Comment (osanseviero) [inline .units/unit1/glossary.mdx:3]:
```suggestion
# Glossary [[glossary]]

This is a community-created glossary. Contributions are welcomed!

### Markov Property
```

File: .units/unit1/glossary.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- In this method, the Policy is learned directly. 
+- Will map each state to the best corresponding action at that state. Or a probability distribution over the set of possible actions at that state.
+
+### Value-based Methods:
+- Another approach to solving RL problems.
+- Here, instead of training a policy, we train a **value function** that maps each state to the expected value of being in that state.
+
+Contributions are welcomed :hugs:
+
+If you want to improve the course, you can [open a Pull Request.](https://github.com/huggingface/deep-rl-class/pulls)
```
Comment (osanseviero) [inline .units/unit1/glossary.mdx:44]:
```suggestion
If you want to improve the course, you can [open a Pull Request.](https://github.com/huggingface/deep-rl-class/blob/main/.units/unit1/glossary.mdx)
```

**Review by osanseviero — APPROVED** [review]
Very cool! Thanks a lot for the contribution :fire:

**Comment by simoninithomas** [comment]
Awesome thanks! I merge 🤗

**Pull request merged**